ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -35,67 +37,65 @@ ScrollView {
             id: timer
         }
 
-        Rectangle {
+        Button {
             id: newMessagesBox
-            visible: state !== "hidden"
-            color: Style.current.secondaryBackground
+            height: 32
+            width: nbMessages.width + arrowImage.width + 2 * Style.current.halfPadding + (nbMessages.visible ? 5 : 0)
```

Comment:
Should we be checking `nbMessages.width` instead of hardcoding it to `5`? What if the user has `2,995` messages to check?
```suggestion
            width: nbMessages.width + arrowImage.width + 2 * Style.current.halfPadding + (nbMessages.visible ? nbMessages.width : 0)
```

Reply:
`5` here is the size of the right padding. ` nbMessages.width` is already taken into account at the start of the line. 
I'll replace by  a padding property just to solve the confusion anyway